examples/http-client-oauth2-client-credentials-grant-type/http_client_oauth2_client_credentials_grant_type.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - OAuth2 client credentials grant type
 
-A client, which is secured with OAuth2 client credentials grant type can be used to connect to a secured service. The client is enriched with the `Authorization: Bearer <token>` header by passing the `http:OAuth2ClientCredentialsGrantConfig` for the `auth` configuration of the client.
+An `http:Client` can be secured with OAuth2 client credentials grant type authentication. This is achieved by enriching each request from the client with the `Authorization: Bearer <token>` header. The required configurations for the OAuth2 client credentials grant type can be specified in the `auth` field as the `http:OAuth2ClientCredentialsGrantConfig` record. Use this to communicate with the service, which is secured with OAuth2 authentication.
```

Comment:
oauth2 is used for authorization (not for authentication)

Reply:
Fix via [30a1a64](https://github.com/ballerina-platform/ballerina-distribution/pull/3793/commits/30a1a645001e203cffbbca292c0b0a5d7d34f335)